lib/project.rb

```diff
@@ -145,12 +145,15 @@ def check_run_termination
       @github.handle_cancel
     end
 
-    def init_workspace
-      unless @options.common.workspace_path.nil?
-        @workspace_path = @options.common.workspace_path
-        return
-      end
+    def session
+      @workspace_path = @options.test.session
+
+      raise AutoHCKError, 'Workspace path does not exist could not load session' unless File.directory?(@workspace_path)
 
+      @logger.info("Loading session from #{@workspace_path}")
+    end
+
+    def create_session
```

Comment:
It says `create_session` but it also creates the workspace directory, which is confusing.